tests/integration/cmor/_fixes/native6/test_era5.py

```diff
@@ -1,14 +1,23 @@
 """Tests for the fixes of ERA5."""
+import datetime
+
 import iris
 import numpy as np
 import pytest
 from cf_units import Unit
 
-from esmvalcore.cmor._fixes.native6.era5 import (AllVars, Evspsbl, Zg,
-                                                 get_frequency)
+from esmvalcore.cmor._fixes.native6.era5 import (
+    AllVars,
+    Evspsbl,
+    Zg,
+    get_frequency,
+)
 from esmvalcore.cmor.fix import Fix, fix_metadata
 from esmvalcore.cmor.table import CMOR_TABLES
 
+COMMENT = ('Contains modified Copernicus Climate Change Service Information '
+           f'{datetime.datetime.now().year}')
```

Comment:
Is this comment originally created somewhere else in ESMValTool? Or is it already in the cube we get from ECMWF? 
If the former, could we add a pointer to that original comment?
If the latter, how do you know the year will be *this* year

Reply:
It is added by ESMVaTool here: https://github.com/ESMValGroup/ESMValCore/blob/d6b20f242e5435ae580e5aab33eeefd3d173f7f7/esmvalcore/cmor/_fixes/native6/era5.py#L329

Reply:
Yes, that's how they asked that you cite their data when we implemented it, though I cannot find back that information anywhere now.